domains/certificates/Query.API/API.AppTests/Testcontainers/RabbitMqContainer.cs

```diff
@@ -0,0 +1,41 @@
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using API.RabbitMq.Configurations;
+using Testcontainers.RabbitMq;
+using Xunit;
+
+namespace API.AppTests.Testcontainers;
+
+public partial class RabbitMqContainer : IAsyncLifetime
+{
+    private readonly global::Testcontainers.RabbitMq.RabbitMqContainer testContainer;
+
+    [GeneratedRegex("\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}:(\\d+)", RegexOptions.CultureInvariant, matchTimeoutMilliseconds: 1000)]
```

Comment:
Is it possible to put this regex in a var so it is more readable?

Reply:
See changes in 296d402e11ef17cbe387d86aea306a84756eca60 
The regex pattern is given in an attribute and I cannot use a variable here.

Reply:
Side note: SonarCloud reported a code smell for not using the GeneratedRegex-attribute. Before that I created the Regex in the `Options` get-method and had the option to extract the pattern into a variable; but that resulted in a code smell.